src/main/java/com/selfxdsd/selfweb/api/WalletsApi.java

```diff
@@ -237,7 +237,11 @@ public WalletsApi(final User user) {
                 if(wallet.active()) {
                     activated = wallet;
                 } else {
+                    final Wallet previousActive = wallets.active();
                     activated = wallets.activate(wallet);
+                    if(previousActive.type().equals(Wallet.Type.FAKE)){
+                        wallets.remove(previousActive);
+                    }
```

Comment:
@criske You don't need to remove the wallet manually. Just say ``wallet.activate()`` and this method will automatically delete the fake wallet if ``wallet`` is ``StripeWallet`` or other.

Reply:
@amihaiemil I'll add that alert in this PR, should be an easy fix, and I'll leave puzzle to make first created payment method active by default. 

About `wallet.activate()`, that was my initial thought too, but I couldn't find where we delete the fake wallet.
In `ProjectWallets#activate` we delegate to storage, but in storage we only disable it.
https://github.com/self-xdsd/self-storage/blob/c68d6fa0b40cf282bbc1e5fe1e3ff85ec754b411/src/main/java/com/selfxdsd/storage/SelfWallets.java#L164
Unless I'm missing something :D

Reply:
@criske the FakeWallet is deleted inside ``StripeWallet.activate()``. Now that you mention it, it will be easier if we delete it inside ``ProjectWallets.activate(Wallet)``... we'll do this change later, it's not urgent now (we won't have other types of wallets anytime soon).